### ISSUE TITLE
Add Header styling / styled components

### ISSUE THREAD

**Issue by riddlew** [CommunityFocus/CommunityFocus#48]
## What
Create the styled component(s) and styling for the header component (#45) and its child components (Logo and WelcomeMessage). This issue depends on #45 being finished, so this cannot be done until that issue is complete.

Please reference the Figma design at https://www.figma.com/file/aOxY6JeMesN7fq72tZsbD9/Untitled?type=design&t=ojy9KXiE1A3V8cHZ-0 for an idea of how this component will look (design subject to change). This component is located at the top of the screen.

![Header](https://github.com/CommunityFocus/CommunityFocus/assets/6940264/04f8325e-0353-4370-a734-fd395ecf059c)

## Details
- Add styling and styled component(s) to the Header component and child components (Logo and WelcomeMessage).
- No testing required as of right now, this will be handled in a separate issue.
